v2/internal/frontend/desktop/darwin/frontend.go

```diff
@@ -463,6 +463,17 @@ func processURLRequest(_ unsafe.Pointer, wkURLSchemeTask unsafe.Pointer) {
 	requestBuffer <- webview.NewRequest(wkURLSchemeTask)
 }
 
+//export processURLRequestStop
+func processURLRequestStop(_ unsafe.Pointer, wkURLSchemeTask unsafe.Pointer) {
+	r := webview.NewRequest(wkURLSchemeTask)
+	requestID, err := r.URL()
```

Comment:
The `requestID` shouldn't just be the `r.URL()` , it should also contain the `r.Method()`.